scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,157 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    group_role TEXT;
+BEGIN
+    group_role := cartodb._CDB_Group_GroupRole(group_name);
+    PERFORM cartodb._CDB_Group_CreateGroup_API(current_database(), group_name, group_role);
+    EXECUTE 'CREATE ROLE "' || group_role || '" NOLOGIN;';
+END
+$$ LANGUAGE PLPGSQL VOLATILE;
+
+-- Drops group and everything that role owns
+-- TODO: LIMITATION: in order to drop a role all its owned objects must be dropped before.
+-- Right now this is done with DROP OWNED, which can only be done by a superadmin.
+-- Not even the role creator can drop the role and the objects it owns.
+-- All group owned objects by the group are permissions.
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_DropGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    cdb_group_role TEXT;
+BEGIN
+    cdb_group_role := cartodb._CDB_Group_GroupRole(group_name);
+    PERFORM cartodb._CDB_Group_DropGroup_API(current_database(), group_name);
+    EXECUTE 'DROP OWNED BY "' || cdb_group_role || '"';
+    EXECUTE 'DROP ROLE IF EXISTS "' || cdb_group_role || '"';
+END
+$$ LANGUAGE PLPGSQL VOLATILE;
+
+-- Renames a group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_RenameGroup(old_group_name text, new_group_name text)
+    RETURNS VOID AS $$
+BEGIN
+    EXECUTE 'ALTER ROLE "' || cartodb._CDB_Group_GroupRole(old_group_name) || '" RENAME TO "' || cartodb._CDB_Group_GroupRole(new_group_name) || '"';
```

Comment:
Sure. Just rewrote all string concatenations at Groups and at the new Organization functions.